content/quickstart/kubernetes/faq.md

```diff
@@ -1,120 +1,176 @@
-    changed. Helm 2 does not.
-
- *  Pulumi and Helm 2 both use the official [Kubernetes Go client library][client-go] to talk to the
-    Kubernetes API server.
+    changed.
 
  *  Helm 2 parameterizes YAML templates using Go templates, a textual replacement engine. Go templates
     are not guaranteed to generate syntactically-correct YAML. Pulumi exposes a rich, multi-language
     SDK, with strong typing to catch errors, and a pre-deployment validation step to catch errors
     before you run the program.
 
+## Can I use my existing YAML manifests and Helm Charts?
+
+Pulumi has integration support for Kubernetes YAML manifests and Helm Charts. This
+support affords users with facilities to jumpstart their Pulumi experience by
+bridging the gap between existing Kubernetes workloads, and net new Pulumi
+code.
+
+This integration support eases the transition to Pulumi by making
+getting started with Pulumi straight-forward
+
+### Examples
+
+##### YAML Manifest:
+
+```typescript
+import * as k8s as "@pulumi/kubernetes";
+
+const myapp = new k8s.yaml.ConfigFile("app.yaml");
+```
+
+##### Helm Chart:
+```typescript
+import * as k8s as "@pulumi/kubernetes";
+
+// Deploy a version of the stable/wordpress chart.
+const wordpress = new k8s.helm.v2.Chart("wpdev", {
+    repo: "stable",
+    version: "2.1.3",
+    chart: "wordpress"
+});
+```
+
+See the [pulumi/examples][examples] repo for more details.
+
+## Does the Pulumi Service see my credentials in the kubeconfig file?
+
+No. Only the `pulumi` CLI tool running locally in the user's client can read the kubeconfig file or execute
+resource operations on the cluster.
+
+The service is used to coordinate multiple user updates, so they don't update the
+same resources concurrently, and stores the state history of your deployments.
+
+If you ask Pulumi to boot up a cluster (_e.g._, as our [EKS example][eks] shows), then it is
+possible to retrieve a kubeconfig file _for that cluster only_. In this case, the kubeconfig file is
+stored as part of the state history of rollouts -- but they are considered opaque state by the
+Pulumi service. The service **never** uses these values for any reason whatsoever.
+
+More over, the contents of the kubeconfig files are not sensitive for managed clusters
+on the major cloud providers, as they defer to their authentication mechanisms for their respective CLI.
+
+## Does the Pulumi Service see my cloud provider credentials?
+
+No. The `pulumi` CLI tool runs locally in the user's client. This allows
+Pulumi to transparently rely on the underlying cloud provider's CLI and authentication
+mechanisms. The service does not read, use, depend, or store any credentials.
+
+## Can I use Pulumi without the pulumi.com service?
+
+Yes. The service performs several functions that are useful, such as coordinating so that
+multiple users don't simultaneously update the same resources, state
+checkpointing & management, and team auth/identity.
```

Comment:
I think there are a few other notable features. Wouldn't hurt to include a comprehensive list from https://www.pulumi.com/pricing/